### ISSUE TITLE
Allow helm diff failures

### ISSUE THREAD

**Issue by jkroepke** [roboll/helmfile#1600]
Follow up from: https://github.com/roboll/helmfile/issues/1595#issuecomment-732886578

If a Chart wants to install CRD and Resources, helm diff may fail. 

As I know a common workaround in helmfile is to disable the validation (https://github.com/roboll/helmfile/issues/1124 and. https://github.com/roboll/helmfile/pull/1373). 

But validation is always good and protect agist spell issues or wrong indents, disable it might be the last resort. On Vanilla helm (3.4.1), I didn't need to disable the validation. 

In this case an option would be great to just allow failures from helm diff since most in the time the failures are non critical.

I know, I could suppress diff completely. 

But I could I suppress diff only on installations? Or what about to disable validation only for helm diff but still enabled for the helm upgrade command?

Thanks

**Comment by mumoshu** [comment]
@jkroepke Hey! In #1618, I've added `disableValidationOnInstall` which I believe is exactly the first option you proposed. Would you mind giving it a try?

**Comment by jkroepke** [comment]
`--skip-diff-on-install` looks more interest for me.

**Comment by mumoshu** [comment]
@jkroepke Yeah- I usually recommend using command-line flags for operational concerns and use YAML config for anything else. So I can also suggest using `--skip-diff-on-install` depending on your situation.

**Comment by DjinNO** [comment]
It's can be also because of
```
experiments {
  manifest = true
 }
```
in Helm Terraform provider
